ejp-rd-vp/vp-dp-frontend
# #1 - Add homepage

**What's in the PR**
* Homepage That looks exactly like the provided mockup

**How to test manually**
* Check design 
* Check all urls in the homepage ('/sources'  & '/imprint' won't work beacuse the feature is not implemented yet)
* close #1
